test/e2e/antreapolicy_test.go

```diff
@@ -1853,12 +1863,14 @@ func testAuditLoggingBasic(t *testing.T, data *TestData) {
 	assert.Equalf(t, true, strings.Contains(stdout, "test-log-acnp-deny"), "audit log does not contain entries for test-log-acnp-deny")
 
 	destinations := []string{"z/a", "z/b", "z/c"}
-	srcIP, _ := podIPs["x/a"]
+	srcIPs, _ := podIPs["x/a"]
 	for _, d := range destinations {
-		dstIP, _ := podIPs[d]
-		// The audit log should contain log entry `... Drop <ofPriority> SRC: <x/a IP> DEST: <z/* IP> ...`
-		pattern := `Drop [0-9]+ SRC: ` + srcIP + ` DEST: ` + dstIP
-		assert.Regexp(t, pattern, stdout, "audit log does not contain expected entry for x/a to %s", d)
+		dstIPs, _ := podIPs[d]
+		for i := 0; i < len(srcIPs); i++ {
+			// The audit log should contain log entry `... Drop <ofPriority> SRC: <x/a IP> DEST: <z/* IP> ...`
+			pattern := `Drop [0-9]+ SRC: ` + srcIPs[i] + ` DEST: ` + dstIPs[i]
```

Comment:
I guess we assume that the v4 address and the v6 address always appear at the same position in the slice?

Reply:
Changed.